File: Oke_server/oke/src/main/java/com/bnuz/oke/dto/VoteData.java
package com.bnuz.oke.dto;

import com.bnuz.oke.entity.Vote;
import com.bnuz.oke.entity.VoteChoice;

import java.util.List;

/**
 * VoteData
 *
 * @author handsome
 * @date 2020年 06月02日 23:32:49
 */
public class VoteData {

	private Vote vote;

	private List<VoteChoice> voteChoiceList;

	public Vote getVote() {
		return vote;
	}

	public void setVote(Vote vote) {
		this.vote = vote;
	}

	public List<VoteChoice> getVoteChoiceList() {
		return voteChoiceList;
	}

	public void setVoteChoiceList(List<VoteChoice> voteChoiceList) {
		this.voteChoiceList = voteChoiceList;
	}

	@Override
	public String toString() {
		return "VoteData{" +
				"vote=" + vote +
				", voteChoiceList=" + voteChoiceList +
				'}';
	}
}

File: Oke_client/Oke_teacher/Oke_teacher/Dto/VoteData.cs
using Oke_teacher.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Oke_teacher.Dto
{
    /// <summary>
    /// 投票信息数据类
    /// </summary>
    [Serializable]
    public class VoteData
    {
        //投票信息
        public Vote vote { set; get; }
        //投票的选项
        public List<VoteChoice> voteChoiceList { set; get; }
    }
}
